stdlib/src/collections/list.mojo

```diff
@@ -548,6 +611,14 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
         Returns:
             The underlying data.
         """
+
+        @parameter
+        if Self.sbo_enabled:
+            if self._sbo_is_in_use():
+                # Someone else is going to free the pointer, the data in the buffer will be invalid
+                # We have to give a pointer that will outlive this object.
+                # YOLO, could do better
```

Comment:
**Suggestion** This workaround seems reasonable in the short term, but can we replace this with a `TODO` or `FIXME` comment (unfortunately `YOLO: XXX` isn't something folks are in the habit of grepping for to search for work that needs doing 😛)

Reply:
haha, sure thing, this is still in draft, I wouldn't let that go to the main branch :)